.vscode/settings.json

```diff
@@ -6,5 +6,9 @@
   "editor.tabSize": 2,
   "editor.formatOnType": true,
   "editor.formatOnPaste": true,
-  "editor.formatOnSave": true
+  "editor.formatOnSave": true,
+  "i18n-ally.localesPaths": [
+    "lang",
+    "functions/src/lang"
```

Comment:
Locale files are in  "lang" directory.

Reply:
Definitely! my bad

Reply:
Oh, sorry. There was a lang directory above. I overlooked it.
You can also add this setting. 

Thanks.